app/Filament/Resources/LocationResource.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources;
+
+use App\Domain\Locations\Models\Location;
+use App\Filament\Resources\LocationResource\Pages;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Tables;
+use Filament\Tables\Table;
+
+class LocationResource extends Resource
+{
+    protected static ?string $model = Location::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';
+
+    protected static ?int $navigationSort = 2;
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->schema([
```

Comment:
Looks good so far I assume you are just leaving these here for time being.

Reply:
Yeah i do need to fill out the form here for this. but I guess with the table items for instance I should probably remove empty arrays right.